### DISCUSSION TITLE
“pot.app”已损坏，无法打开。 你应该将它移到废纸篓。

### DISCUSSION THREAD

**Discussion by elarbor** [pot-app/pot-desktop#921] [General]
<img width="369" alt="image" src="https://github.com/user-attachments/assets/c8b9a587-d45e-4699-a071-3ab01d0f11ca">

**Comment by xtyuns** [comment]
参考常见问题：[应用没有签名 (MacOS)](https://pot-app.com/docs/faq.html#%E5%BA%94%E7%94%A8%E6%B2%A1%E6%9C%89%E7%AD%BE%E5%90%8D-macos)
